wp-rocket.php

```diff
@@ -65,6 +65,38 @@
 
 require WP_ROCKET_INC_PATH . 'compat.php';
 
+if ( version_compare( PHP_VERSION, '5.3', '<' ) ) {
```

Comment:
Faster (same goes for line 297):
```php
version_compare( PHP_VERSION, '5.3' ) < 0
```

Also, you use `phpversion() >= '5.4'` on line 166, maybe use the same notation everywhere?